### ISSUE TITLE
Core Tools v3 version should reset to 3.0.0 at GA

### ISSUE THREAD

**Issue by ahmedelnably** [Azure/azure-functions-core-tools#1657]
Core Tools v3 version should reset to 3.0.0 at GA

**Comment by brettsam** [comment]
Are you referring to the version here? https://github.com/Azure/azure-functions-core-tools/releases/tag/3.0.1740. We've always used the build numbers in our versioning, rather than going `3.0.1`, `3.0.2`, etc. 

I don't disagree with going back to 3.0.0, but we should come up with a versioning strategy to make these clear that they're previews of 3.0.0.

**Comment by ahmedelnably** [comment]
Agree, we have https://github.com/Azure/azure-functions-core-tools/issues/1249 to figure out a new versioning schema that makes it more logical

**Comment by ahmelsayed** [comment]
I used to up the minor version whenever I add new commands to the cli. But I don't think everyone was following that. The last bit in the version was a-forever-increasing build number.
